.github/workflows/test.yml

```diff
@@ -15,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         cache_nonce: [ 1 ]
-        nim_version: [ 1.6.6 ]
+        nim_version: [ 1.6.20, stable ]
```

Comment:
Hmm, I don't know if we're ready for 2.0 yet? 

Also, I think the latest version in `nimbus-build-system` is 1.6.21, we should probably set it to that as well?

Reply:
Aren't odd numbered patches considered unstable?